src/context-menu/context-menu.m.css

```diff
@@ -0,0 +1,13 @@
+.root {
```

Comment:
comments please

Reply:
I've removed these CSS files, as you mentioned in another comment these styles are handled by the popup now and I just failed to remove the unused file